### PR TITLE
Migrate to new organization

### PR DESCRIPTION
**Pull request by Simply007**
### Motivation

Which issue does this fix? Fixes #`issue number`

If no issue exists, what is the fix or new feature? Were there any reasons to fix/implement things that are not obvious?

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

**Comment by IvanKiral** [comment]
DELIVERY SDK:
- [ ] There are some `Kontent` instead of `Kontent.ai` ( comment suggestions in at least Readme are coming later with full review)
- [ ] Missing Javadocs (AsyncCache, Delivery client) - add or maybe create Issue?
- [ ] Restructure folders -> sometimes it is hard to implementation of interface (optional, also candidate for new issue)
- [ ]  Code is mostly good from my perspective, some comments are coming with full review later

**Comment by IvanKiral** [comment]
Delivery-sdk-generators:
- [ ] Code looks good, only there are some only Kontent mentions. Although there are only comments they can be changed with replace

**Comment by IvanKiral** [comment]
sample-app-android:
- [ ] There is a Floating Action button registered in `ArticlesFragment.java` it is not visible in the app though.
- [ ] There are some not changed Kontent -> Kontent.ai

**Comment by IvanKiral** [comment]
sample-app-spring-boot:
- [ ] Change fonts to work sans
- [ ] Article.html has twice <!DOCTYPE html>
- [ ] Date in articles has weird format
- [ ] Linked items in rich text element are not working.. In article `coffee berages explained` clicking on `so rich as today` should navigate us to the `which brewing fits you?` article

**Comment by IvanKiral** [comment]
Parent:
- There is a kontent-delivery folder with only empty readme

**Review by IvanKiral — CHANGES_REQUESTED** [review]

**Comment by Simply007** [comment]
I haven's found any old MIT liense in java files.

**Comment by Simply007** [comment]
> DELIVERY SDK:
> 
> * [x]  There are some `Kontent` instead of `Kontent.ai` ( comment suggestions in at least Readme are coming later with full review)
> * [ ]  All .java files does not have updated MIT license on the top of the file
I haven't found any old licences
> * [ ]  Missing Javadocs (AsyncCache, Delivery client) - add or maybe create Issue?
Added for asynccache and delivery client - create issue for the rest
> * [ ]  Restructure folders -> sometimes it is hard to implementation of interface (optional, also candidate for new issue)
Create issue if you want to 
> * [x]   Code is mostly good from my perspective, some comments are coming with full review later

File: delivery-sdk/src/main/java/kontent/ai/delivery/Asset.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
  * SOFTWARE.
  */
 
-package kentico.kontent.delivery;
+package kontent.ai.delivery;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
```
[the remark this answers is not in the document]
Comment (Simply007) [reply delivery-sdk/src/main/java/kontent/ai/delivery/Asset.java:28]: I would leave it this way - or create a separate issue if you like it more 👍🏼

**Comment by Simply007** [comment]
> sample-app-spring-boot:
> 
> * [x]  Change fonts to work sans
> * [ ]  Article.html has twice
I don't understand
> * [x]  Date in articles has weird format
> * [x]  Linked items in rich text element are not working.. In article `coffee berages explained` clicking on `so rich as today` should navigate us to the `which brewing fits you?` article

File: delivery-sdk/src/main/java/kontent/ai/delivery/CacheManager.java [hunk 2]
```diff
@@ -45,23 +45,23 @@
...
      */
     JsonNode get(final String url);
 
     /**
      * Cache a response from the KontentDelivery API.
      *
-     * @param url the URL that was used to retrieve the response from the KontentDelivery API.
-     * @param jsonNode the JsonNode created from the response from the Kontent Delivery API.
+     * @param url the URL that was used to retrieve the response from the Kontent.ai Delivery API.
+     * @param jsonNode the JsonNode created from the response from the Kontent.ai Deliver API.
```
Comment (IvanKiral) [inline delivery-sdk/src/main/java/kontent/ai/delivery/CacheManager.java:64]: Delivery

File: delivery-sdk/src/main/java/kontent/ai/delivery/CacheManager.java [hunk 3]
```diff
@@ -45,23 +45,23 @@
...
      * Retrieve an earlier cached response from the KontentDelivery API.
      *
-     * @param url The url that would be used to retrieve the response from Kontent Delivery API.
+     * @param url The url that would be used to retrieve the response from Kontent.ai Delivery API.
      * @return JsonNode response or null if no value is available in the cache for the given url.
      */
     JsonNode get(final String url);
 
     /**
      * Cache a response from the KontentDelivery API.
```
Comment (IvanKiral) [inline delivery-sdk/src/main/java/kontent/ai/delivery/CacheManager.java:61]: kontent.ai delivery api

**Review by IvanKiral — APPROVED** [review]

**Review by IvanKiral — APPROVED** [review]

**Pull request merged**